lib/matplotlib/axes/_axes.py

```diff
@@ -7,12 +7,12 @@
 import numpy as np
 from numpy import ma
 
-import matplotlib.category  # Register category unit converter as side-effect.
+#import matplotlib.category  # Register category unit converter as side-effect.
```

Comment:
Please revert commenting out these imports. As the comment states, we need them for their side-effects.

Reply:
Thanks for your feedback! I will make the changes as requested.